src/services/exit-logs/verifier.ts

```diff
@@ -0,0 +1,239 @@
+import { LRUCache, makeLogger } from '../../lib/index.js'
+
+import { ethers } from 'ethers'
+
+import { funcDTO, txDTO } from './dto.js'
+import { ExecutionApiService } from '../../services/execution-api/service.js'
+
+const ORACLE_FRAME_BLOCKS = 7200
+
+export type VerifierService = ReturnType<typeof makeVerifier>
+
+export const makeVerifier = (
```

Comment:
Let's split this mega function into smaller

Reply:
This looks like a minor improvement, I suggest doing it in conjunction with TW
https://linear.app/lidofi/issue/ORC-355/validatorejector-tw-preparations